_inc/client/writing/masterbar.jsx

```diff
@@ -40,7 +40,7 @@ export const Masterbar = withModuleSettingsFormHelpers(
 						<p>
 							{ __(
 								'The WordPress.com toolbar replaces the default WordPress admin toolbar. ' +
-									'It offers one-click access to notifcations, your WordPress.com ' +
+									'It offers one-click access to notifications, your WordPress.com ' +
 									'profile and your other Jetpack and WordPress.com websites. ' +
 									'You can also catch up on the sites you follow in the Reader.'
```

Comment:
>You can also catch up on the sites you follow in the Reader.

Do Jetpack customers know what "Reader" is? Should this be "WordPress.com Reader" here?

Reply:
I think that creates too much repetition. I think the solution would probably be to link it through to the reader itself.